src/assets-importer/index.js

```diff
@@ -23,12 +26,21 @@ function fetchFromFigma(endpoint) {
   }).then(res => res.json());
 }
 
+/**
+ * @param  {string} {name - the file name
+ * @param  {RegExp} regex} - the regex used to normalize the file
```

Comment:
Does this format actually work for named params? Did you try to generate the docs and see how does it look like?

I usually use:
```js
/**
 * @param  {String} paramsObject.name - the file name
 * @param  {RegExp} paramsObject.regex - the regex used to normalize the file
 */
```

But I've also read that `$0` gives better output, but I've never tested it:
```js
/**
 * @param  {String} $0.name - the file name
 * @param  {RegExp} $0.regex - the regex used to normalize the file
 */
```

Reply:
Generated with JSDoc Live Preview on VSCode, looks like this:
![image](https://user-images.githubusercontent.com/2879127/107520143-6272b480-6bb1-11eb-94d4-ead58e817416.png)
Using $0 looks exactly like the last one. The params in the method title don't show up, so I think it didn't work.

Reply:
Ah seems like `JSDoc Live Preview` doesn't use the same structuring as the official parser, the official one, nests the params inside, instead of repeating `paramsObject`.

In that case I'd prefer to go for removing the `{` and `}` in the param naming, because it will just cause confusion and it will give a wrong definition in the editor:
![Screen Shot 2021-02-10 at 4 45 05 PM](https://user-images.githubusercontent.com/30146019/107533993-c8fecf00-6bbf-11eb-85af-c73a78715a40.png)
Because the name of the param will be `{name` instead of `name`.

Let's proceed this way until we agree on a way to document named params.